src/sparseml/transformers/finetune/data/data_helpers.py

```diff
@@ -218,4 +220,32 @@ def get_custom_datasets_from_path(path: str, ext: str = "json") -> Dict[str, str
                 if dir_dataset:
                     data_files[dir_name] = dir_dataset
 
+    return transform_dataset_keys(data_files)
+
+
+def transform_dataset_keys(data_files: Dict[str, Any]):
+    """
+    Transform dict keys to `train`, `val` or `test` for the given input dict
+    if matches exist with the existing keys
```

Comment:
So if data-files contains keys : `train_sft, train_something_else` transformation would not take place?

but if we only have one of the two it would? Could we add that in the docstring?